topic/topic.go

```diff
@@ -92,6 +97,16 @@ func (t *topic) SetConsumerServices(value []ConsumerService) Topic {
 	return &newt
 }
 
+func (t *topic) Version() int {
```

Comment:
Hm how is the version useful? Seems this is something that shouldn't be exposed to the users of this library no?

Reply:
Mainly for logging reasons, so that user can log that it's processing topic with version X

Reply:
Not sure if users of this library need to know that the topic has versions, but okay.